README.md

```diff
@@ -132,26 +136,42 @@ griddb_fdw might return an error when DML is executed.
 
 For example, container on GridDB has 3 columns. The 1st column is "c1" as integer, the 2nd is "c2" as float, and the 3rd is "c3" as text.  
 Schema must be created as
+
 ```
 CREATE FOREIGN TABLE ft1 (c1 integer, c2 float, c3 text) SERVER griddb_svr;
 ```
 
 You should not execute following queries.  
 
 Types are not match.
+
 ```
 CREATE FOREIGN TABLE ft1 (c1 text, c2 float, c3 integer) SERVER griddb_svr;
 ```
 There is unknown column.
 Even if unknown column is dropped, griddb cannot access ft1 correctly.
+
 ```
 CREATE FOREIGN TABLE ft1 (c0 integer, c1 integer, c2 float, c3 text) SERVER griddb_svr;
 ALTER FOREIGN TABLE ft1 DROP COLUMN c0;
 ```
+#### Don't support ON CONFLICT.

```

Comment:
* This function seems to exist since around 2015, so I'd like to change the expression a little. (something like from 'introduces' to 'has'?)

Reply:
I understand your comment. We change 'introduces' to 'has'.